quetz/rest_models.py

```diff
@@ -65,8 +66,10 @@ class ChannelBase(BaseModel):
     private: bool = Field(True, title="channel should be private")
     size_limit: Optional[int] = Field(None, title="size limit of the channel")
     ttl: int = Field(36000, title="ttl of the channel")
-    mirror_channel_url: Optional[str] = Field(None, pattern="^(http|https)://.+")
-    mirror_mode: Optional[MirrorMode] = Field(None)
+    mirror_channel_url: Optional[Union[URLType, List[URLType]]] = Field(
```

Comment:
I feel like the goal of backward compatibility does not really warrant the confusion the `singular/plural` tighrope walk will induce here. Quetz is currently in major version `0`, we can afford to break backward compatibility in this very minor. Let's just go to a list and return a meaningful error in case someone tries to use the old way.